### PR TITLE
add some questions in FAQ

### PR DESCRIPTION
**Pull request by wlwgcdxc**

### CODE REVIEW

File: doc/zh-cn/FAQ.md [hunk 1]
```diff
@@ -65,4 +65,26 @@ export DOCKER_API_VERSION=1.38
...
 
-NB-IoT是一种网络制式，和2/3/4G类似，带宽窄功耗低。NB-IoT支持基于TCP的MQTT通信协议，因此可以使用NB-IoT卡连接百度云物接入，部署OpenEdge应用和BIE云管理通信。但国内三大运营商中，电信对他们的NB卡做了限制，仅允许连接电信的云服务 IP，所以目前只能使用移动NB卡和联通NB卡连接百度云服务。
\ No newline at end of file
+NB-IoT是一种网络制式，和2/3/4G类似，带宽窄功耗低。NB-IoT支持基于TCP的MQTT通信协议，因此可以使用NB-IoT卡连接百度云物接入，部署OpenEdge应用和BIE云管理通信。但国内三大运营商中，电信对他们的NB卡做了限制，仅允许连接电信的云服务 IP，所以目前只能使用移动NB卡和联通NB卡连接百度云服务。
+
+**问题8**: var/run/openedge.sock: address already in use
+
+删除var/run/openedge.sock，重启Openedge。
+
+**问题9**: Openedge支持数据计算后将计算结果推送Kafka吗？
```
Comment (ludanfeng) [inline doc/zh-cn/FAQ.md]: Openedge -> OpenEdge

File: doc/zh-cn/FAQ.md [hunk 2]
```diff
@@ -65,4 +65,26 @@ export DOCKER_API_VERSION=1.38
...
+
+**问题11**: Openedge函数计算框架，支持函数的多运行实例吗？如何配置？
+
+函数计算框架会根据实时的计算负载启动多个运行实例为计算提供算力，核心配置如下：
+```
+instance: 函数实例配置项
+  min: 默认值：0，最小值：0，最大值：100，最小函数实例数
+  max: 默认值：1，最小值：1，最大值：100，最大函数实例数
+```
+更加具体的配置参见：[函数计算模块配置](https://github.com/baidu/openedge/blob/master/doc/zh-cn/tutorials/Config-interpretation.md#函数计算模块配置)
```
Comment (ludanfeng) [inline doc/zh-cn/FAQ.md]: 问题11移除，文档里已经写清楚的问题，直接把对应的文档发给用户，FAQ里不要写，容易两边不一致。

File: doc/us-en/FAQ.md [hunk 3]
```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
 
 **Question 7**: How does BIE access the NB-IOT network?
```
Comment (kuailehaibin) [inline doc/us-en/FAQ.md]: BIE --> OpenEdge

Comment (kuailehaibin) [reply doc/us-en/FAQ.md]: How does OpenEdge connect to NB-IoT network?

File: doc/us-en/FAQ.md [hunk 4]
```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
...
 
-NB-IoT is a network standard similar to 2/3/4G with low bandwidth and low power consumption. NB-IoT supports TCP-based MQTT protocol, so you can use NB-IoT card to connect to Baidu Cloud IotHub, deploy OpenEdge application and communicate with BIE Cloud Management Suite. However, among the three major operators in China, Telecom have imposed whitelist restrictions on their NB cards, and only allow to connect to Telecom Cloud Service IP. Therefore, only Mobile NB cards and Unicom NB cards can be used to connect to Baidu Cloud Service.
\ No newline at end of file
+NB-IoT is a network standard similar to 2/3/4G with low bandwidth and low power consumption. NB-IoT supports TCP-based MQTT protocol, so you can use NB-IoT card to connect to Baidu Cloud IotHub, deploy OpenEdge application and communicate with BIE Cloud Management Suite. However, among the three major operators in China, Telecom have imposed whitelist restrictions on their NB cards, and only allow to connect to Telecom Cloud Service IP. Therefore, only Mobile NB cards and Unicom NB cards can be used to connect to Baidu Cloud Service.
```
Comment (kuailehaibin) [inline doc/us-en/FAQ.md]: BIE --> [BIE](https://cloud.baidu.com/product/bie.html)

File: doc/us-en/FAQ.md [hunk 5]
```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
...
+
+**Question 8**: var/run/openedge.sock: address already in use
+
+Remove var/run/openedge.sock and restart Openedge.
+
+**Question 9**: Does openedge support to push data to kafka?
+
+For support, you can use the function framework to write a function that is responsible for subscribing messages from the Hub and writing them to Kafka one by one. You can also customize the module, which subscribes to the Hub and then writes kafka in bulk.
```
Comment (kuailehaibin) [inline doc/us-en/FAQ.md]: For support, you can use the local function module to write a [Python script](./customize/How-to-write-a-python-script-for-python-runtime.md) that is responsible for subscribing messages from the local Hub module and writing them to Kafka service. Besides, you can also [develop a customize module](./customize/How-to-develop-a-customize-module-for-openedge.md), which subscribes message from the local Hub module and then writes it to Kafka.

File: doc/us-en/FAQ.md [hunk 6]
```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
...
+
+**Question 10**: What are the ways to change openedge configurations? Can I only make configuration changes through the BIE Cloud Management Suite?
```
Comment (kuailehaibin) [inline doc/us-en/FAQ.md]: openedge --> OpenEdge

File: doc/us-en/FAQ.md [hunk 7]
```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
...
+
+Currently, we recommend changing configurations through the BIE Cloud Management Suite, but you can also manually change the configuration file on the core device and then restart openedge to take effect.
+
+**Question 11**: Does the openedge function framework support multiple running instances of function? How to configure?
```
Comment (kuailehaibin) [inline doc/us-en/FAQ.md]: Does the OpenEdge function framework support multiple running instances of function? If support, how to configure?

File: doc/us-en/FAQ.md [hunk 8]
```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
...
+
+The function framework would start multiple running instances based on the real-time computing load. The core configuration is as follows:
+```
+instance: function instance configuration
+  min: The default value is `0`, means the minimum number of function instance. And the minimum configuration allowed to be set is `0`, the maximum configuration allowed to be set is `100`.
+  max: The default value is `1`, means the maximum number of function instance. And the minimum configuration allowed to be set is `1`, the maximum configuration allowed to be set is `100`.
+```
+For more function configuration: [Local Function Configuration](https://github.com/baidu/openedge/blob/master/doc/us-en/tutorials/Config-interpretation.md#local-function-configuration)
```
Comment (kuailehaibin) [inline doc/us-en/FAQ.md]: For more function configuration: [Local Function Configuration](./tutorials/Config-interpretation.md#local-function-configuration)

File: doc/zh-cn/FAQ.md [hunk 9]
```diff
@@ -65,4 +65,26 @@ export DOCKER_API_VERSION=1.38
 
 **问题7**: BIE如何接入NB-IoT?
```
Comment (kuailehaibin) [inline doc/zh-cn/FAQ.md]: BIE --> OpenEdge

File: doc/zh-cn/FAQ.md [hunk 10]
```diff
@@ -65,4 +65,26 @@ export DOCKER_API_VERSION=1.38
...
\ No newline at end of file
+NB-IoT是一种网络制式，和2/3/4G类似，带宽窄功耗低。NB-IoT支持基于TCP的MQTT通信协议，因此可以使用NB-IoT卡连接百度云物接入，部署OpenEdge应用和BIE云管理通信。但国内三大运营商中，电信对他们的NB卡做了限制，仅允许连接电信的云服务 IP，所以目前只能使用移动NB卡和联通NB卡连接百度云服务。
+
+**问题8**: var/run/openedge.sock: address already in use
+
+删除var/run/openedge.sock，重启Openedge。
+
+**问题9**: Openedge支持数据计算后将计算结果推送Kafka吗？
+
+支持，您可以使用函数计算框架，编写一个函数，负责从Hub订阅消息，并将消息逐个写入Kafka。您也可以自定义模块，该模块向Hub订阅消息，然后批量写入Kafka。
```
Comment (kuailehaibin) [inline doc/zh-cn/FAQ.md]: 支持，您可以使用函数计算框架，[编写一个函数](./customize/How-to-write-a-python-script-for-python-runtime.md)，负责从Hub模块订阅消息，并将消息逐个写入Kafka。您也可以[开发一个自定义模块](./customize/How-to-develop-a-customize-module-for-openedge.md)，该模块向Hub模块订阅消息，然后批量写入Kafka服务。

File: doc/zh-cn/FAQ.md [hunk 11]
```diff
@@ -65,4 +65,26 @@ export DOCKER_API_VERSION=1.38
 
 **问题7**: BIE如何接入NB-IoT?
 
-NB-IoT是一种网络制式，和2/3/4G类似，带宽窄功耗低。NB-IoT支持基于TCP的MQTT通信协议，因此可以使用NB-IoT卡连接百度云物接入，部署OpenEdge应用和BIE云管理通信。但国内三大运营商中，电信对他们的NB卡做了限制，仅允许连接电信的云服务 IP，所以目前只能使用移动NB卡和联通NB卡连接百度云服务。
\ No newline at end of file
+NB-IoT是一种网络制式，和2/3/4G类似，带宽窄功耗低。NB-IoT支持基于TCP的MQTT通信协议，因此可以使用NB-IoT卡连接百度云物接入，部署OpenEdge应用和BIE云管理通信。但国内三大运营商中，电信对他们的NB卡做了限制，仅允许连接电信的云服务 IP，所以目前只能使用移动NB卡和联通NB卡连接百度云服务。
```
Comment (kuailehaibin) [inline doc/zh-cn/FAQ.md]: BIE --> [BIE](https://cloud.baidu.com/product/bie.html)

File: doc/zh-cn/FAQ.md [hunk 12]
```diff
@@ -65,4 +65,26 @@ export DOCKER_API_VERSION=1.38
...
+
+**问题8**: var/run/openedge.sock: address already in use
+
+删除var/run/openedge.sock，重启Openedge。
+
+**问题9**: Openedge支持数据计算后将计算结果推送Kafka吗？
+
+支持，您可以使用函数计算框架，编写一个函数，负责从Hub订阅消息，并将消息逐个写入Kafka。您也可以自定义模块，该模块向Hub订阅消息，然后批量写入Kafka。
+      
+**问题10**: Openedge配置更改的方式有哪些？只能通过云端管理套件进行配置更改吗？
```
Comment (kuailehaibin) [inline doc/zh-cn/FAQ.md]: Openedge --> OpenEdge

File: doc/zh-cn/FAQ.md [hunk 13]
```diff
@@ -65,4 +65,26 @@ export DOCKER_API_VERSION=1.38
...
+
+目前，我们推荐通过云端管理套件进行配置定义和下发，但您也可以手动更改核心设备上的配置文件，然后重启Openedge使之生效。
```
Comment (kuailehaibin) [inline doc/zh-cn/FAQ.md]: Openedge --> OpenEdge

**Pull request merged**